src/ISA/ISA/Update.fs

```diff
@@ -1,5 +1,17 @@
 ﻿namespace ARCtrl.ISA.Aux
 
+module List = 
```

Comment:
make this internal? So we never flood namespaces.

Reply:
Will first make a static member out of it. I think we need a place for more complex operations, that do not necessarily belong to ISA.Spreadsheet or ISA.Json.